screen.py

```diff
@@ -546,12 +547,23 @@ def remove_subscription(self, panel_name):
 
     def check_dpms_state(self):
         state = functions.get_DPMS_state()
-        if state == functions.DPMS_State.Off and "screensaver" not in self._cur_panels:
-            logging.info("### Creating screensaver panel")
-            self.show_panel("screensaver", "screensaver", "Screen Saver", 1, False)
-        elif state == functions.DPMS_State.On and "screensaver" in self._cur_panels:
-            logging.info("### Remove screensaver panel")
-            self._menu_go_back()
+
+        visible = self.get_property("visible")
+        if state == functions.DPMS_State.Off and visible:
+            logging.info("DPMS State Off -> Hiding")
+            self.hide()
+            os.system("xsetroot  -cursor_name  watch")
+            self.touch_ready = False
+        elif state == functions.DPMS_State.On and not visible:
+            if self.touch_ready:
+                logging.info("DPMS State On -> Showing KlipperScreen")
+                self.show()
+                os.system("xsetroot  -cursor_name  arrow")
```

Comment:
This shouldn't necessarily be arrow. If you look at line 143, it could be arrow or blank.

Reply:
xsetroot only affects the root window, the cursor definitions that you mention are respected inside KlipperScreen but not during hide (that's why you can see the watch cursor), the arrow or watch that are in here will be shown on windows that are not KS, and no unfortunately with this method you cannot set it blank, that needs to be done during xinit using the -nocursor flag, xsetroot can only set names in /usr/include/X11/cursorfont.h

Reply:
it that there is a very old workaround of using an xbm, i will try that

Reply:
hi guys, just saw the changes made in the code. im kinda hornored, enthusiastic you guys took the proposal and run with it :D 

just wanna pass some extra info. 
i made some extra logging in my version for logging after x time and noticed that when the screen is blank the state is suspend and not off (one off the main reasons that i used not on)
log:
`2021-11-30 00:09:03,604 [screen.py:check_state()] - log line after checking 600.421985 seconds:  checked 1199 times, DPM_state Suspend, window is hidden`


(just for info the code that makes the log)
```
        state = functions.get_DPMS_state()
        global counter, time_start, count
        log_text = ["FAIL", "ON", "Standby", "Suspend", "OFF"]
        log_visible =["hidden", "visible"]
        isvisible = self.get_property("visible")
        time_interval = time.monotonic()
        if time_start == 0.0:
            logging.info("First run, checked %d times, DPM_state %s, window is %s", count, log_text[state + 1],
                         log_visible[isvisible])
            time_start = time_interval
            count = 0
        elif (time_interval - time_start) > 600.0:
            logging.info("log line after checking %f seconds:  checked %d times, DPM_state %s, window is %s",
                         time_interval - time_start, count, log_text[state + 1], log_visible[isvisible])
            time_start = time_interval
            count = 0
```

Reply:
no problem, i didn't play with the timeout value yet. i've got it set to 500 now, will test 200 and lower

Reply:
Hi, i tested some values, lower then 200 introduces the touch issue again (for me). this also may be personal, some maybe want a more responsive system others not

just a quote from #207 
> have already seen the "status information". it was helpful for testing, but that's true. this will unnecessarily flood the journal. I commented it out
> 
> I also played around with the delay-time. 500ms seems good to me (250ms is to quick for me, misclicks happens)

i think for now 500 ms is fine enough